packages/flutter_tools/templates/app/linux.tmpl/my_application.cc.tmpl

```diff
@@ -1,6 +1,9 @@
 #include "my_application.h"
 
 #include <flutter_linux/flutter_linux.h>
+#ifdef GDK_WINDOWING_X11
```

Comment:
Nit: conditional includes should be in their own block after all other includes.